ui/main.qml

```diff
@@ -24,6 +26,10 @@ StatusWindow {
 
     property bool appIsReady: false
 
+    readonly property var featureFlags: typeof featureFlagsRootContextProperty !== undefined ? featureFlagsRootContextProperty : null
```

Comment:
Perhaps we could move the creation of the whole `FeatureFlagsStore` one level up to here if we really need to access it earlier

Reply:
Oh, and the flag itself should also be added to the store as well